src/commands/git-metadata/upload.ts

```diff
@@ -117,8 +125,13 @@ export class UploadCommand extends Command {
         metricsLogger.logger.increment('gitdb.success', 1)
         this.logger.info(`${this.dryRun ? '[DRYRUN] ' : ''}Successfully synced git DB in ${elapsed} seconds.`)
       } catch (err) {
-        console.log('error writing to git db')
-        this.logger.warn(`Could not write to GitDB: ${err}`)
+        if (!this.isTargetingGov()) {
+          console.log('error writing to git db')
+          this.logger.warn(`Could not write to GitDB: ${err}`)
```

Comment:
Why both a `console.log` and a `this.logger.warn`?

Reply:
(I understand it was there before, but maybe it was an oversight?)

Reply:
Good catch it looks like something I missed in a previous PR, thanks